packaging/rpm/Makefile

```diff
@@ -30,8 +33,10 @@ srpm: build_prepare
 		-r $(MOCK_CONFIG) \
 		--define "__version $(VERSION)" \
 		--define "__release $(BUILD_NUMBER)" \
+		--enable-network \
```

Comment:
If you ever plan on building inside of docker then you need to pass in `--old-chroot` to mock commands to tell docker to use `chroot` in place of `systemd-nspawn` (because `systemd-nspawn` wouldn't work within a container, privileged or otherwise).

Reply:
Thank you, good to know!